frontend/src/views/QuizCreateView.vue

```diff
@@ -0,0 +1,306 @@
+<template>
+  <div class="container">
+    <!-- 문제 내용 입력 컴포넌트 -->
+    <div class="content-field">
+      <label for="content" class="content">문제 내용</label>
+      <textarea type="text" :value="content" placeholder="문제를 입력하세요" @input="handleContentInput"
+        class="content-input" />
+    </div>
+    <hr />
+
+    <!-- 선택지 입력 컴포넌트들 -->
+    <div v-for="(option, index) in options" :key="index" class="option-field">
+      <div class="option-input-wrapper">
+        <!-- 선택지 입력란 -->
+        <input type="radio" :name="'option'" :value="index + 1" v-model="answer" :id="'option' + index" hidden />
+        <label :for="'option' + index" class="answer-button" :class="{ 'selected': answer === index + 1 }">
```

Comment:
- `JavaScript`의 `Template literals` 기능을 사용하면 `+`연산자 없이 문자열 내에 변수를 넣을 수 있어
이렇게 하는게 더 직관적으로 이해하기 편하고, 작성하기도 편해
- 동적 바인딩을 사용하지 않을거라면 앞에 `:`를 붙이지 않고 쓸 수 있어

```suggestion
        <input type="radio" name="option" :value="index + 1" v-model="answer" :id="`option${index}`" hidden />
        <label :for="`option${index}`" class="answer-button" :class="{ 'selected': answer === index + 1 }">
```

참고
[MDN Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
적용했습니다.